src/_playground/Routes.js

```diff
@@ -387,9 +387,9 @@ export class Routes extends Component {
                                     inputValue={this.state.query}
                                     onChange={this.onChangeHandler} />
                             </div>
-                            <nav className='frDocs-Nav'>
+                            <SideNav className='frDocs-Nav'selectedId=''>
```

Comment:
I think the `selectedId` should always be set here.  Initially, it should be for the "Home" page, but then should get reset each time the page changes.